yang/ietf-ucl-acl.yang

```diff
@@ -44,14 +44,72 @@ module ietf-ucl-acl {
       "RFC XXXX: A Policy-based Network Access Control";
   }
 
-  feature match-on-endpoint-group {
+  identity group-acl-type {
+    if-feature "group";
+    base acl:acl-base;
+    description
+      "An ACL that matches based on an abstract group identity,
+       which may be within the NVO3 header extension, but may not
+       correspond to any field in the packet header.";
+  }
+  identity mixed-ipv4-group-type {
+    if-feature "mixed-ipv4-group";
+    base acl:ipv4-acl-type;
+    base uacl:group-acl-type;
+    description
+      "An ACL that contains a mix of entries that match on fields
+       in IPv4 headers and abstract group identities.  Matching on 
+       Layer 4 header fields may also exist in the list.";
```

Comment:
which list?

Reply:
It follows RFC8519. Or how about this: s/in the list/in the ACL entries/?

Reply:
I would then change to "ACEs".

Reply:
fixed.